### DISCUSSION TITLE
Is there a way to only search for the exact search term?

### DISCUSSION THREAD

**Discussion by dirk68-work** [Pagefind/pagefind#575] [Q&A]
Hi, I use the default UI because I don't know a lot of JavaScript.
When searching for „Laser“ on my site there are results shown for „lassen“ and „lässt“. And some are ranked higher than a result containing „Laserbox“.
Is there an easy way to only search for exact occurences of „laser“?

**Comment by bglw** [comment]
👋 

Currently there isn't a way to get that kind of exact search, no — Pagefind internally stems words down to their roots for indexing, which means a word like `laser` is indexed as `las`, which is then giving more permissive matches for you.

There is some work underway to expose more controls around result scoring, and one of those controls will influence how highly those extensions like `lassen` rank — hoping to have that out in the near future on the next Pagefind release.
